docs/docs/features/low-power-states/low-power-states.mdx

```diff
@@ -0,0 +1,253 @@
+
+#### KScan sideband behavior
+
+<Tabs groupId="advanced-methods" defaultValue="direct" className="secrettabs">
+  <TabItem value="direct" label="Direct Pin">
+    <SidebandDirect />
+  </TabItem>
+  <TabItem value="matrix" label="Matrix Integrated Pin">
+    <SidebandMatrix />
+  </TabItem>
+  <TabItem value="wakeup" label="Wakeup-only Keyswitch">
+    <Tabs groupId="kscan-type" defaultValue="mkscan" className="secrettabs">
+      <TabItem value="dkscan" label="Direct GPIO kscan">
+        <SidebandWakeupDirect />
+      </TabItem>
+      <TabItem value="mkscan" label="Matrix kscan">
+        <SidebandWakeupMatrix />
+      </TabItem>
+    </Tabs>
+  </TabItem>
+</Tabs>
+
+Here are the properties for the kscan sideband behaviors node:
+
+<Tabs groupId="advanced-methods" defaultValue="direct" className="secrettabs">
+  <TabItem value="direct" label="Direct Pin">
+    <PropsSidebandDirect />
+    <PropsSidebandChildren />
+  </TabItem>
+  <TabItem value="matrix" label="Matrix Integrated Pin">
+    <PropsSidebandMatrix />
+    <PropsSidebandChildren />
+  </TabItem>
+  <TabItem value="wakeup" label="Wakeup-only Keyswitch">
+    <Tabs groupId="kscan-type" defaultValue="mkscan" className="secrettabs">
+      <TabItem value="dkscan" label="Direct GPIO kscan">
+        <PropsSidebandDirect />
+      </TabItem>
+      <TabItem value="mkscan" label="Matrix kscan">
+        <PropsSidebandMatrix />
+      </TabItem>
+    </Tabs>
+  </TabItem>
+</Tabs>
+
+<Tabs groupId="advanced-methods" defaultValue="direct" className="secrettabs">
+  <TabItem value="direct" label="Direct Pin"></TabItem>
+  <TabItem value="matrix" label="Matrix Integrated Pin">
+  You also need to update the `zmk,kscan` chosen value to point to the new kscan instance:
+
+```
+/ {
+    chosen {
+        ...
+        zmk,kscan = &side_band_behavior_triggers;
+        ...
+    };
+};
+```
+
+  <SoftOffWaker />
+  </TabItem>
+  <TabItem value="wakeup" label="Wakeup-only Keyswitch">
+    <Tabs groupId="kscan-type" defaultValue="mkscan" className="secrettabs">
+        <TabItem value="dkscan" label="Direct GPIO kscan"></TabItem>
+        <TabItem value="mkscan" label="Matrix kscan"><SoftOffWaker /></TabItem>
+    </Tabs>
+  </TabItem>
+</Tabs>
+
+Finally, we will list the `wakeup_scan` device in an additional configuration section so that the ZMK soft off process knows it needs to enable this device as part of the soft off processing so it can wake the keyboard from soft off when pressed:
+
+```
+/ {
+    soft_off_wakers {
+        compatible = "zmk,soft-off-wakeup-sources";
+        wakeup-sources = <&wakeup_scan>;
+    };
+};
+```
+
+Here are the properties for the node:
+
+- The `compatible` property for the node must be `zmk,soft-off-wakeup-sources`.
+- The `wakeup-sources` property is a [phandle array](../../config/index.md#devicetree-property-types) pointing to all the devices that should be enabled during the shutdown process to be sure they can later wake the keyboard.
+
+:::tip
```

Comment:
Should we put this under "wakeup-only" tab? It is essentially wakeup-only but with any key, isn't it?

Reply:
We certainly could and it would make sense. I quite like it where it is because it gives a bit better intuition of what `wakeup-sources` does, and I don't think it's causing any confusion at that location.